### PR TITLE
Integrate npm command with package

### PR DESCRIPTION
**Pull request by digitalsadhu**
Use package command for packaging under NPM namespace. 
As per an earlier discussion, this PR moves the npm publishing functionality from the npm command to the package command. 

To publish to the npm namespace, use the `--npm` flag:

```sh
eik package --npm
```

### CODE REVIEW

File: test/integration/alias.test.js [hunk 1]
```diff
@@ -36,12 +36,23 @@ beforeEach(async (done, t) => {
         key: 'change_me',
     }).run();
 
-    const publishCmd = `${eik} npm scroll-into-view-if-needed 2.2.24
-        --token ${token}
-        --server ${address}
-        --cwd ${folder}`;
+    const assets = {
+        name: 'scroll-into-view-if-needed',
```
Comment (stipsan) [inline test/integration/alias.test.js:40]: Oh wow, who uses `scroll-into-view-if-needed` in real apps?? *Cringe* 😉

**Review by stipsan — APPROVED** [review]
LGTM 👍

**Review by trygve-lie — APPROVED** [review]
lgtm :+1:

File: test/integration/alias.test.js (hunk 1 above)
Comment (digitalsadhu) [reply test/integration/alias.test.js:40]: I heard it's a popular module by a rock star programmer. ;)

**Pull request merged**